src/backends/fixerio/index.js

```diff
@@ -91,7 +92,7 @@ class FixerIoBackend {
    * @param {String} params.destination_ledger The URI of the destination ledger
    * @param {String} params.source_currency The source currency
    * @param {String} params.destination_currency The destination currency
-   * @returns {Object}
+   * @returns {Promise.<Object>}
```

Comment:
ah wait, it doesn't return a promise to an object, it yields an object. will change this back.